### PR TITLE
Several small tweaks to docs / makefile

### PR DESCRIPTION
**Pull request by SgtCoDFish**
I spotted a few things while experimenting with the sample-external-issuer on a dev day; this PR addresses some of the smaller non-functional changes which are easy to make.

**Note for reviewer**: I've intentionally split this PR into several smaller commits. This PR may be best reviewed on a per-commit basis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,28 @@
 # sample-external-issuer
 
-This is an example of an [External Issuer] for cert-manager.
+External issuers extend [cert-manager](https://cert-manager.io/) to issue certificates using new APIs and services
```
Comment (jakexks) [inline README.md:3]: if I were to nitpick here, the word new doesn't seem right.
```suggestion
External issuers extend [cert-manager](https://cert-manager.io/) to issue certificates using APIs and services
```

**Review by jakexks — APPROVED** [review]
Looks good, I only have the nitpickiest of nitpicks. Feel free to merge if you want anyway
/hold
/lgtm

**Review by wallrj — APPROVED** [review]
Thanks Ash,

/lgtm
/approve

File: README.md (hunk 1 above)
Comment (jakexks) [reply README.md:3]: Post review comment: The suggested diff looks dumb because the sentence continues on the next line

Comment (SgtCoDFish) [reply README.md]: yeah, I think you're right, it has a feeling of "brand new" rather than "new to cert-manager". replaced :+1:

**Pull request merged**